web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1152,6 +1152,9 @@ private int executeStatement(boolean silent, String user, String pw, String sql,
 			// TODO how to get the driver for the other dbs...
 			if (wizardModel.databaseConnection.contains("mysql")) {
 				Class.forName("com.mysql.cj.jdbc.Driver").newInstance();
+			} else if (wizardModel.databaseConnection.contains("postgres")) {
+				Class.forName("org.postgresql.Driver").newInstance();
+				replacedSql = replacedSql.replaceAll("`", "\"");
```

Comment:
Is the replacedSql.replaceAll needed for postgresql?

Reply:
Currently, if you remove this you'll get the error :
![Screenshot from 2020-06-05 23-41-59](https://user-images.githubusercontent.com/39997970/83909786-0b495400-a787-11ea-97c1-316cc94bf4cd.png)